### PR TITLE
Create a module with common attributes

### LINKED ISSUE #1558: Create shared CSS classes module
While CSS properties should belong to respective components, sometimes they can be common among different components. For example, a "button" could be an `a` or a `button`; or some set of classes does not really belong to any component like defining parent's properties which affect the children.

The proposed module name - `components::classes`. Example class:

```rust
fn container() -> Attrs {
  class![C.relative, C.group]
}
```
This would be used in components which need a tooltip or a popup or both.

Here could be various aliases, for example, colours.

### PR DESCRIPTION
**Pull request by ip1981**
Closes #1558.

Signed-off-by: Igor Pashev <pashev.igor@gmail.com>

---
This change is [Reviewable](https://reviewable.io/reviews/whamcloud/integrated-manager-for-lustre/1563)

### CODE REVIEW

**Comment by ip1981** [comment]
I am thinking about lazy static too:

```rust
lazy_static! {
    pub(crate) static ref CONTAINER: Attrs = class![C.relative, C.group];
}
```

**Comment by jgrund** [comment]
I'd consider using lazy static sparingly, in this case a fn seems just as good and more straightforward.

File: iml-gui/crate/src/components/attrs.rs [hunk 1]
```diff
@@ -0,0 +1,16 @@
+/**
```
Comment (jgrund) [inline iml-gui/crate/src/components/attrs.rs]: Hrrm, does this show up as a module-level doc comment?

**Comment by ip1981** [comment]
> Yes, it can be done as the global object, but I forsee the need to pass some additional parameters there, and then it will need to be converted to the function back.
> 
 Oh, no :) Everything here is supposed to be dumb simple.

**Pull request merged**